README.md

```diff
@@ -260,7 +260,7 @@ go get -u github.com/alecthomas/chroma/cmd/chroma
 
 - Quite a few lexers, for various reasons (pull-requests welcome):
     - Pygments lexers for complex languages often include custom code to
-      handle certain aspects, such as Perl6's ability to nest code inside
```

Comment:
This sentence now no longer makes sense?

Reply:
I only removed the perl6 part and missed the regular expressions part.
Do you mean the whole "Pygments lexers for complex languages often include custom code to..." part. or just that "regular expressions part"?

Reply:
I just mean it now reads "handle certain aspects regular expressions", which doesn't make sense. Maybe just revert this, or did you mean to change it to Raku?